Automattic/wp-calypso
# Signup reskin: Update UI for domain search results

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:

This PR implements only the UI changes for the domain search results. Other UI changes are implemented in subsequent PRs.

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

For the record:
Parent PR: https://github.com/Automattic/wp-calypso/pull/51504 and root PR: https://github.com/Automattic/wp-calypso/pull/50512

**With search results:**

<img width="1238" alt="Screenshot 2021-03-31 at 5 21 48 PM" src="https://user-images.githubusercontent.com/1269602/113151534-cda44300-9252-11eb-9829-4acaff8b4d7d.png">

**Search results with on-hover effect:**

<img width="1031" alt="Screenshot 2021-03-31 at 5 21 58 PM" src="https://user-images.githubusercontent.com/1269602/113151638-ea407b00-9252-11eb-932c-809190203cef.png">

**Search results with the ".com" tld filter applied:**

<img width="1232" alt="Screenshot 2021-03-31 at 5 22 12 PM" src="https://user-images.githubusercontent.com/1269602/113151690-f6c4d380-9252-11eb-9725-45eab695ae37.png">


#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
*  Proceed to the domains step and verify that the UI matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536). 
* Start over and verify that control group experience is unaffected
